tox.ini

```diff
@@ -4,14 +4,14 @@ skip_missing_interpreters = true
 
 [testenv:pylint]
 deps =
-   git+https://github.com/PyCQA/astroid@master
+   git+https://github.com/rogalski/astroid@infer-open-calls
```

Comment:
I'll revert it after merge. For now on, Py3 inference of `open` result is still lacking.